src/components/appBar/AppBar.tsx

```diff
@@ -54,8 +56,7 @@ const AppBar = ({ title, showBackButton = false, topPanelHeight }: Props) => {
   };
 
   const goBack = () => {
-    if (history.length > 2) history(-1);
-    else history(ROUTES.DASHBOARD);
+    history(-1);
```

Comment:
@Kasmadei  Why do we no longer need to check the logic of `history.length`? Is it ok?

Reply:
No need to explicitly check the length of the history stack before navigating back. "history(-1)" function call itself handles the navigation back operation. Also it was like this before, but am sure we can remove that check